### PR TITLE
rename NOT_ENGLAND variable to NI_REGION_CODE 

### PR DESCRIPTION
**Pull request by JacobHarrisonONS**
# What and why?
Fix a bit of confusion around the following bit of code 

{{ // if not not england is england. welcome to ras-rm.
if (!NOT_ENGLAND.equals(region)) {
region = "";
}}}

rename NOT_ENGLAND so it can make more sense

# How to test?

# Jira

https://jira.ons.gov.uk/browse/RAS-769

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/response/casesvc/representation/action/NotifyModel.java [hunk 1]
```diff
@@ -115,8 +115,8 @@ public Classifiers build() {
             return new Classifiers(actionType, "", "", "", "");
           }
 
-          // if not not england is england. welcome to ras-rm.
-          if (!NOT_ENGLAND.equals(region)) {
+          // if NOT NI then England
```
Comment (SteveScorfield) [inline src/main/java/uk/gov/ons/ctp/response/casesvc/representation/action/NotifyModel.java:118]: I'd put England/Wales just to be 100% clear

**Review by SteveScorfield — APPROVED** [review]
Looks good. One minor alteration the comments but that's it.

File: src/main/java/uk/gov/ons/ctp/response/casesvc/representation/action/NotifyModel.java (hunk 1 above)
Comment (JacobHarrisonONS) [reply src/main/java/uk/gov/ons/ctp/response/casesvc/representation/action/NotifyModel.java:118]: okay - ideally we wouldnt need the comment but its already been confusing

Comment (SteveScorfield) [reply src/main/java/uk/gov/ons/ctp/response/casesvc/representation/action/NotifyModel.java]: Agreed

**Pull request merged**